gps/gps.cpp

```diff
@@ -0,0 +1,190 @@
+#include "gps.h"
+#include <string>
+#include <stdio.h>
+#include <filesystem>
+#include <sstream>
+#include <iostream>
+
+using namespace std;
```

Comment:
Is this a style we want to use? https://stackoverflow.com/questions/1452721/whats-the-problem-with-using-namespace-std